@types/slate-react/index.d.ts

```diff
@@ -0,0 +1,68 @@
+declare module 'slate-react' {
+  import type { TEditor } from '@commercetools-uikit/rich-text-utils/src/editor.types';
+  import { Component } from 'react';
+
+  type TValue = TEditor['value'];
+  type TWrapBlock = ReturnType<TEditor['unwrapBlock']>;
+  type TNode = Parameters<NonNullable<TEditor['query']>>[1];
+  type TEditorProps = {
+    autoFocus: unknown;
+    id: unknown;
+    name: unknown;
+    onFocus: unknown;
+    onBlur: unknown;
+    disabled: unknown;
+    readOnly: unknown;
+    value: unknown;
+    options: unknown;
+    onChange: unknown;
+    plugins: unknown;
+    renderEditor: unknown;
```

Comment:
I’ve decided to leave these types as they are. I guess copy-pasting them here won’t give us any more type safety, will bloat this declaration file and we have a task for updating this package anyway (#2120)

Reply:
Since the `Editor` component is using the `RichTextBody` component internally and we already have defined [the types](https://github.com/commercetools/ui-kit/blob/main/packages/components/inputs/rich-text-utils/src/editor.types.ts) in that package, I was wondering if we import those types here instead of creating this file for consistency.
_(we would need to export them from the `rich-text-utils` package)_

Reply:
I'm wondering if I understand correctly, what you are suggesting is copy-pasting `TEditor` type from `rich-text-utils` to `slate-react` type declaration file and finally importing this type from `slate-react` package in both `rich-text-utils` and `rich-text-input`?